packages/integration/.eslintrc.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
```

Comment:
should we organize the folders differently?

I'd love if we had `packages` as the actual storage packages and other folders for supporting packages, examples and integration tests

That way people checking out this repo on github won't get overwhelmed, and it's also easier for us to see which of these are actually published to npm

Reply:
Sounds great

Reply:
done